src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -21,21 +22,27 @@
         return new Person[] {
             new Person(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
                 new Address("Blk 30 Geylang Street 29, #06-40"),
+                BirthDate.of("2000-01-01"),
```

Comment:
Maybe put some variations (different birthdays) between the peoples, so that it is more forward-compatible, like sort by age feature in the future? Same for other data files.